### ISSUE TITLE
Japanese in comments on Google Docs is not recognized

### ISSUE THREAD

**Issue by melink14** [melink14/rikaikun#898]
As far as I can tell, no mouse events are being fired inside of comment boxes, needs more investigation.

**Comment by maawisul** [comment]
The extension worked fine for me on a document in Google Docs when installed from Chrome Webstore, however when I load the unpacked version into chrome, it didn't seem to work anymore.

**Comment by melink14** [comment]
Thanks for the comment.

1. First, to be clear, the part this bug is focused on is just the comments that people can add to Google Docs. For example, if you highlight some text you can attach arbitrary comments like this:
![image](https://user-images.githubusercontent.com/1176550/164352488-20ccad17-663d-4161-8e38-65e2f3008471.png)

Right now even though the main text works, if I hover over my comment in that doc, it doesn't work.

2. I've used unpackaged extension many times when working with Google Docs so it must be a particular problem. Here are some debugging tips:
   a. Go to the unpackaged extension page and ensure 'report errors' is checked. reload the extension and ensure no errors (besides MV2 deprecation notice) are listed in the Errors view.
   b. From the extension page, check that there are no errors printed in the background.html page that you can inspect by clicking on the link there.


Just to confirm, the unpackaged extension works on all other pages but doesn't work on Google Docs?

**Comment by maawisul** [comment]
Yes, the unpackaged extension works fine on all other pages but doesn't work on Google Docs.
